keyboards/xdboards/recon/rules.mk

```diff
@@ -0,0 +1,4 @@
+CONVERT_TO = rp2040_ce
```

Comment:
Converters should **not** be enabled at the keyboard level.

(Any existing use in the repo has either already been removed, or is in the process of being removed.)

Reply:
Understood, thank you for the insight! Is it okay to move to the default keymap?